gulp/full.js

```diff
@@ -1,3 +1,15 @@
 var gulp = require('gulp');
+var gulpSequence = require('gulp-sequence');
 
-gulp.task('full', ['build', 'tslint', 'all-tests']);
+var isMaster = require('./util/is-master');
+var isTagAdded = require('./util/is-tag-added');
+
+gulp.task('build-and-tests', ['build', 'tslint', 'all-tests']);
+gulp.task('build-and-tests-and-publish', gulpSequence('build-and-tests', 'publish'));
+
+if (isMaster() && isTagAdded()) {
```

Comment:
I think `gulp publish` should be executed in this case. And add `cdnjs` build task under `publish` lately